optimum/gptq/quantizer.py

```diff
@@ -112,6 +113,8 @@ def __init__(
             max_input_length (`Optional[int]`, defaults to `None`):
                 The maximum input length. This is needed to initialize a buffer that depends on the maximum expected input length.
                 It is specific to the exllama backend with act-order.
+            cache_block_outputs (`bool`, defaults to `False`):
+                Whether to cache block outputs to reuse as inputs for the succeeding block.
```

Comment:
Maybe add a comment saying that it might not work for some models such as chatgml.